pandas/core/groupby/groupby.py

```diff
@@ -350,12 +350,12 @@ class providing the base-class of operations.
 
 See Also
 --------
-%(klass)s.groupby.apply : Apply function func group-wise
-    and combine the results together.
+%(klass)s.groupby.apply : Apply function ``func`` group-wise and combine
+    the results together.
 %(klass)s.groupby.aggregate : Aggregate using one or more
     operations over the specified axis.
-%(klass)s.transform : Transforms the Series on each group
-    based on the given function.
+%(klass)s.transform : Call ``func`` on self producing a %(klass)s with
```

Comment:
I would think that this should be about e.g. DataFrame.groupby.transform, rather than DataFrame.transform.

Reply:
So this is the doc string for DataFrame.groupby.transform. So in the "See Also" section, it makes sense to me that it references the ungrouped DataFrame.transform.

Reply:
Ahh, whoops, thanks! In that case, may want to specify that transform operates on the columns rather than self.

Reply:
Sorry, let me be more specific. This is a template doc-string that is reused for both [DataFrame|Series].groupby.transform. Hence the "klass" templating. In master right now, the "see also" section specifically talks about Series, which doesn't make sense for DataFrame. So my PR here is to fix that. The text I inserted, is taken verbatim from the template for [DataFrame|Series].tansform. I think that's the reason it's being a little vague, and avoiding verbiage that would imply dimensionality.

Reply:
Thanks - make sense. It does seem to me desirable to improve this (namely, it seems to me to be misleading in the DataFrame case), but okay for this PR.